Dockerfile

```diff
@@ -4,13 +4,19 @@ MAINTAINER Dmitry Mozzherin
 ENV LAST_FULL_REBUILD 2016-10-24
 
 RUN apt-get update && \
-    apt-get install -y software-properties-common && \
-    apt-add-repository ppa:brightbox/ruby-ng && \
+    apt-get install -y software-properties-common curl && \
+    apt-add-repository ppa:brightbox/ruby-ng
+RUN apt-get install apt-transport-https -y && \
```

Comment:
@dimus good call; I've addressed this in 0178086822494a0d9e1f29a32ad33040a5459b74